index.html

```diff
@@ -10,6 +10,21 @@
 <body>
   <header>
     <a href="#" id="my-logo">Farida's Logo</a>
+    <img src="images/home.png" alt="description" id="home"/>
   </header>
-      </body>
+
+  <section id="intro">
+    <section id="text">
+      <h1>Farida Hamid</h1>
+      <p>Software Engineer and consultant to educational platform since March 2020.</p>
+      <h3>let's connect</h3>
+      <ul id="contacts">
+        <li><a href="#"><img src="images/vec.png" alt="twitter"></a></li>
+        <li><a href="#"><img src="images/Linkedin-icon.png" alt="Linkedin-icon"></a></li>
+        <li><a href="#"><img src="images/vec-3.png" alt="email"></a></li>
+        <li><a href="#"><img src="images/vec-2.png" alt="github"></a></li>
+      </ul>
+    </section>
+  </section>
+</body>
```

Comment:
- [ ] Currently, your page doesn't match with the Figma design. Please work on that and try to match it with the Figma design. 

Your design             |  Figma Design
:-------------------------:|:-------------------------:
![y1](https://user-images.githubusercontent.com/70483835/159027762-78feece7-660c-4a2f-aa44-5f2e5d9bc0d1.png) | ![f1](https://user-images.githubusercontent.com/70483835/159027788-f0c4527b-9650-4ded-ab25-628791cce8ef.png)

**Optional:**
If you want, you can also use CSS media queries to optimize and make your page responsive.

Reply:
Here's my current design:
<img width="479" alt="Screen Shot 2022-03-19 at 1 05 38 PM" src="https://user-images.githubusercontent.com/33801345/159116765-f62fd991-8117-4e3e-8ec1-9b9d42ce8d7e.png">